decidim/decidim
# Add reminders for pending orders in budgets

#### :tophat: What? Why?
Adds an admin feature to send reminders to users who have not confirmed their orders. Also brings reminder, reminder_record and reminder_delivery models to the core which can be used to create additional reminders for example, for meetings.

#### :pushpin: Related Issues
https://meta.decidim.org/processes/roadmap/f/122/proposals/16243

#### Testing
1. Create pending order that's created more than 24.hours ago
2. Enable background jobs (e.g run delayed_jobs)
3. ```bundle exec rake decidim:reminders:all``` **OR** Go to admin panel -> components -> budgets -> "Send voting reminders"
4. Check mail

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
![image](https://user-images.githubusercontent.com/19709320/146371258-4665547f-a1ba-4d45-a805-7211ef35fce7.png)

:hearts: Thank you!
